content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -0,0 +1,169 @@
+---
+Title: Prometheus integration with Redis Cloud
+linkTitle: Prometheus integration
+description: To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud databases.
+weight: $weight
+alwaysopen: false
+categories: ["RC"]
+---
+
+To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud subscription.
+
+- [Prometheus](https://prometheus.io/) is an open-source systems monitoring and alerting toolkit that can scrape metrics from different sources.
+- [Grafana](https://grafana.com/) is an open-source, feature-rich metrics dashboard and graph editor that can process Prometheus data.
```

Comment:
```suggestion
- [Grafana](https://grafana.com/) is an open-source metrics dashboard and graph editor that can process Prometheus data.
```

Reply:
I would clarify the opening a little: ... you can connect Prometheus to the 'Prometheus compatible endpoint' in the Redis Cloud subscription, then you can connect Grafana to Prometheus for visualization.  You can't connect Grafana directly to the RC metrics endpoint and you technically don't need Grafana, it's just a very common pattern to add Grafana for visualization.